### ISSUE TITLE
XUnit Cannot Run Tests in VS2015 Update 1 with RC1-Final DNX Projects

### ISSUE THREAD

**Issue by ardalis** [xunit/xunit#714]
I'm not sure whether this is an XUnit or a VS issue, but projects that were working fine, like this one:
https://github.com/aspnet/Docs/tree/master/aspnet/testing/unit-testing/sample

no longer run their xunit tests within Visual Studio. I'm using the R# runner but have also tried MSTest. The issue is that it cannot find the DLL it needs. The error is
"Unable to run xUnit.net tests - File not found: {srcpath}\test\artifacts\bin\testproject\testproject.dll"

One issue related to this might be that the project is not producing outputs to disk. However, even when that is turned on, the DLL still isn't in that location, so the error persists.

**Comment by bradwilson** [comment]
Resharper has never been able to run DNX tests.

**Comment by ardalis** [comment]
Ok, then my issue at the moment is how to get the VS test runner to see and run the tests. I have the extension xUnit.net Test Extensions v 1.0.0.101 installed, but test explorer doesn't find any tests. Do I (also) need to pull in a dependency in my test project's <packages> ? /cc @efleming18

**Comment by bradwilson** [comment]
That extension does not help you run tests. That extension helps you generate tests.

Can you repro your issue if you follow these steps exactly? http://xunit.github.io/docs/getting-started-dnx.html

**Comment by flq** [comment]
Hm, I'm not certain, but it may be related. Running...
* Visual Studio 2015+Update1 CE
* A project with project.json file
* In the dependencies section for the "net46" framework a dependency onto _xunit.runner.visualstudio_ (obviously also _xunit_).
* There is also a dnxcore50 section

Shouldn't that be enough for tests to be picked up in the bog standard test explorer of Visual Studio?

I also checked the lower parts of https://xunit.github.io/docs/getting-started-desktop.html - didn't find the temp folder you described, but I licked it clean anyway.

The relevant project is at https://github.com/flq/NMeasure/tree/corefx/NMeasure.Tests - really quite confused here, I don't think I'm doing anything special?
